CHANGELOG.md

```diff
@@ -22,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Parsing errors on parenthesised anonymous methods.
 - Incorrect ordering of edits in the "Separate grouped parameters" quick fix for `GroupedParameterDeclaration`.
 - SonarLint registration of rules that don't support execution in a SonarLint context.
+- Incorrect handling of some type naming in the `PointerName` rule.
```

Comment:
Thanks for updating the changelog! Usually a maintainer would do this before merging (to avoid merge conflicts).

I'd consider this an improvement rather than a fix, as it's making the rule behaviour meaningfully better rather than fixing a quirk. Because of this, I think it could be summed up in a single bullet point under the `Changed` heading instead of two separate points in different headings.